frontend/src/types/IstioConfigList.ts

```diff
@@ -21,11 +21,13 @@ import {
   VirtualService,
   WorkloadEntry,
   WorkloadGroup,
-  IstioObject
+  IstioObject,
+  GroupVersionKind
 } from './IstioObjects';
 import { ResourcePermissions } from './Permissions';
 
 export interface IstioConfigItem {
+  apiVersion?: string;
```

Comment:
`apiVersion` and `kind` should be required. They can probably be their own interface `TypeMeta` that this one `extends`.

```
export interface TypeMeta {
  apiVersion: string;
  kind: string;
}
...
export interface IstioConfigItem extends TypeMeta { ... }
```
I can submit a small PR for this if it'd be helpful?

Reply:
yes please

Reply:
Opened: https://github.com/hhovsepy/kiali/pull/5

Reply:
thanks @nrfox merged